<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serch_sensor</class>
 <widget class="QWidget" name="serch_sensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>431</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>编号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>型号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>生产厂家</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>波长/伸缩值/初值</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>serch</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
